src/components/daily-dropoffs-report.tsx

```diff
@@ -145,13 +145,15 @@ The data model table can't accurately capture:
 `}
         />
       </Accordion>
-      <DailyDropoffsLineChart
-        data={filteredData}
-        width={width}
-        color={issuerColor}
-        funnelMode={funnelMode}
-        scale={scale}
-      />
+      {undefined && (
```

Comment:
I would have made this depend on the `extra` flag like in the other report, but doing that, in combination with how this report splits out agencies vs aggregates them, made this whole report too slow.